app/schema/programModel.js

```diff
@@ -14,8 +14,8 @@ var ProgramSchema = new Schema({
     trim: true,
     required: true
   },
-  host: {
-    type: String,
+  hosts: {
+    type: [String],
     trim: true,
```

Comment:
Not sure how to handle this `trim` line, because while it doesn't make sense with a `String` array, we would ideally like to have each host name trimmed. Try seeing if there's a way to trim each `String` in the array, but if not, just delete this line and we can revisit the concept later